### PR TITLE
add dockerfile for Bodo

### PR DESCRIPTION
**Pull request by rishi1016**

### CODE REVIEW

File: docker/Bodo.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apt-get update && apt-get install -y wget bzip2 git vim make curl\
+	&& wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh \
+	&& chmod +x miniconda.sh\
+	&& ./miniconda.sh -b
+ENV PATH /root/miniconda3/bin/:${PATH}
+
+RUN conda install -y python=3.9 bodo=2021.8 ipyparallel -c bodo.ai -c conda-forge
+
+# Might be useful for long running containers
```
Comment (sahil1105) [inline docker/Bodo.Dockerfile:13]:
```suggestion
```

**Review by sahil1105 — APPROVED** [review]
Looks good to me. Thanks @rishi1016!

**Review by ritwika314 — APPROVED** [review]
(worked for me)

**Comment by sahil1105** [comment]
Should we just upload an image to Dockerhub?
Then people could just do:
```
FROM bodo-ubuntu:latest
```
in their own Dockerfiles.

**Comment by rishi1016** [comment]
> Should we just upload an image to Dockerhub?
> Then people could just do:
> 
> ```
> FROM bodo-ubuntu:latest
> ```
> 
> in their own Dockerfiles.

great idea! let's get Ehsan's feedback on this - thank you for approving :)

**Pull request merged**